src/tv2-common/actions/executeAction.ts

```diff
@@ -2036,8 +2034,6 @@ async function executeActionSelectFull<
 	const externalId = `adlib-action_${context.getHashId(`cut_to_full_${template}`)}`
 
 	const graphicType = config.studio.GraphicsType
-	const prerollDuration =
-		graphicType === 'HTML' ? config.studio.CasparPrerollDuration : config.studio.VizPilotGraphics.OutTransitionDuration
```

Comment:
This was a suspicious line. Why would we use `VizPilotGraphics.OutTransitionDuration` for a preroll duration when we have a `VizPilotGraphics.PrerollDuration`? I think it was a bug that probably was more or less harmless. After the changes, VizPilotGraphics.PrerollDuration` will be used, but Full adlibs are something to keep an eye on when testing.